pkg/controller/quay_io_ci_images_distributor/quay_io_ci_images_distributor.go

```diff
@@ -287,6 +287,15 @@ func imageStreamNameFromImageStreamTagName(nn types.NamespacedName) (types.Names
 	return types.NamespacedName{Namespace: nn.Namespace, Name: colonSplit[0]}, nil
 }
 
+func repoFileGetterWithIgnore(metadata cioperatorapi.Metadata) func(_, _, _ string, _ ...github.Opt) github.FileGetter {
+	// We ignore openshift-priv because it is a private org and thus would lead to 404 without providing authentication information
+	// Moreover, the .ci-operator.yaml file there is a duplication of the one from the public repo
+	if metadata.Org == "openshift-priv" {
```

Comment:
Is a similar issue going to occur for repos in `qe-private-deck` like, for example, `openshift/openshift-tests-private`?